resources/assets/coffee/react/beatmapset-page/header.coffee

```diff
@@ -19,6 +19,14 @@
 el = React.createElement
 
 class BeatmapsetPage.Header extends React.Component
+  toggleFavorite: (e) ->
+    e.preventDefault()
+
+    if _.isEmpty currentUser
+      userLogin.show()
```

Comment:
when I call `userLogin.show e.target`, after logging in and collapsing the menu, the loading overlay shows up and seems not to hide at any point :confused: